package.json

```diff
@@ -18,15 +18,21 @@
     "ci:watea": "yarn install --cwd themes/watea/ --frozen-lockfile --non-interactive"
   },
   "devDependencies": {
+    "@types/eslint": "^7.2.4",
     "babel-eslint": "^10.0.3",
     "css-loader": "^3.2.0",
-    "eslint": "^6.5.1",
+    "eslint": "^6.8.0",
+    "eslint-config-ackama": "^2.0.0",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.14.3",
     "foundation-sites": "^6.5.3",
-    "laravel-mix": "^5.0.0",
+    "laravel-mix": "^5.0.7",
+    "prettier": "^2.1.2",
```

Comment:
Could you add `prettier-plugin-packagejson` too?

Also, don't forget `prettier-config-ackama`, and to add `"prettier": "prettier-config-ackama"` to the `package.json`

Reply:
Can this go into another PR? I'm aware we are adding the old thingie cause these are PRs from august last year that I'm trying to push through in an orderly manner without having things break while doing so

Reply:
I'd prefer if you could add the `prettier-config-ackama` in the same PR you add `prettier`, as otherwise we'll be using the default settings and so have to re-do any formatting from `prettier` when we add the config.

Happy to leave out adding the extra prettier plugin :)

Reply:
That works better I believe